packages/dioxus-oauth/src/firebase.rs

```diff
@@ -127,7 +128,6 @@ impl FirebaseService {
     ) -> Self {
         tracing::debug!("FirebaseService::init: {api_key}, {auth_domain}, {project_id}, {storage_bucket}, {messaging_sender_id}, {app_id}, {measurement_id}");
 
-        #[cfg(feature = "web")]
```

Comment:
it's not a bug.

If you uncomment this, it causes lots of panics on server side due to only-web function of  `initialize_app`

Reply:
@victor138128

Reply:
@hackartists 
Okay, I understand. I will close the PR.